css-highlight-api-1/Overview.bs

```diff
@@ -162,6 +162,11 @@ Creating Custom Highlights</h3>
 	authors can chose between using {{Range}} objects and {{StaticRange}} objects.
 	See [[#range-invalidation]] for more details about this choice and its implications.
 
+	Note: When creating {{Range}} objects for use in [=custom highlights=], it is suggested
+	that authors avoid placing [=range=] endpoints in the middle of characters that are
+	comprised of multiple unicode codepoints. Doing so can create undesirable highlighting
+	effects, ex. highlighting a base character but not its combining mark.
+
```

Comment:
This should probably link the i18n-glossary and use our more formal terminology. Also note that our original ask has to do with quirks of DOM being encoded in UTF-16. Perhaps:

```suggestion
Note: When creating {{Range}} objects for using in [=custom highlights=], authors should avoid placing [=range=] endpoints in the middle of a [=grapheme=], such as when a visible unit of text is comprised of multiple [=Unicode code points=]. Doing so can create undesirable highlighting effects, such as highlighting only part of a Indic syllable. In addition, care should be exercised to avoid placing an endpoint in the middle of a [=supplementary character=].
```